### ISSUE TITLE
Duplicate for Calling saveMapPNGWithCpmpletionBlock(reloadMapImage)

### ISSUE THREAD

**Issue by Cu-Toof** [KiloKilo/WorldMatrix#1]
I have seen the code in WorldMatrixView class:
you are using observer properties:
```
open var mapMatrix: Matrix<WorldCharacteristic>? {
        didSet {
            createMatrixFrames()
            saveMapPNGWithCpmpletionBlock(reloadMapImage)
        }
    }
```
But after updating value for mapMatrix in `func setCharacteristic(...)`, you call saveMapPNGWithCpmpletionBlock(reloadMapImage) again. 
I want to know that calling saveMapPNGWithCpmpletionBlock again is necessary or not.
Sorry, my English skill is not good.
The project is useful for me.
Thank you

**Comment by mekanics** [comment]
Well I think you're right. It's a redundant call. 
I will push a new version to cocoapods during the course of the day.

**Comment by mekanics** [comment]
Has been fixed in #2
